virttest/qemu_vm.py

```diff
@@ -1273,7 +1273,7 @@ def add_vga(devices, vga):
             """Add primary vga device."""
             fallback = params.get("vga_use_legacy_expression") == "yes"
             machine_type = params.get("machine_type", '')
-            parent_bus = {'aobject': 'pci.0'}
+            parent_bus = self._get_bus(params, 'vga')
```

Comment:
`self._get_bus(params, 'vga')` can handle all parent_bus, so we don't need to check machine type below.

@fbq815 I know you never test GUI mode, but if you want, you can try to add a virtio-gpu device in s390 guest, this should work. cc @YongxueHong

Reply:
I do test the virtio-gpu device by using libvirt tools